### ISSUE TITLE
Unauthorized response due to missing headers during cik lookup request

### ISSUE THREAD

**Issue by Paul-Manart** [sec-edgar/sec-edgar#300]
Apologies for the amateur structure, first time reporting a bug. 
Upon running this sample code:
```
from secedgar import filings, FilingType

# 10Q filings for Apple (ticker "aapl")
my_filings = filings(cik_lookup="aapl",
                     filing_type=FilingType.FILING_10Q,
                     user_agent="test.test@hotmail.com")
my_filings.save('output')
```
this error occurs:
requests.exceptions.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

this is due to the fact that the request from _cik_lookup.py_ on line 32 is being sent without headers, thus retrieving a 403 response. If headers with user agent are passed, the cik lookup json is being retrieved instead.

**Comment by snowpearbiggod** [comment]
May I know how to solve it? Should I revise the file cik_lookup.py? Thanks.

**Comment by Paul-Manart** [comment]
I would assume passing a user agent would be sufficient. Since the user must already pass the user agent in the filings() function, I expect it would be easy to pass it along to the 32nd line of cik_lookup.py:
    response = requests.get("https://www.sec.gov/files/company_tickers.json", headers={"User-Agent": "test@hotmail.com"})

**Comment by ilias-ant** [comment]
Since: 
- the issue has been fixed on `master`: https://github.com/sec-edgar/sec-edgar/commit/eb5c3b8de8af70e8511f9e024cca89693cf7af1a
- it is not part of a release yet (maintainers will probably release this soon, i guess)

the most sensible thing one can do for now is to install the library directly on the "bleeding edge" aka master branch with sth like:

`pip install git+https://github.com/sec-edgar/sec-edgar.git@master`

assuming one is familiar with potential risks.

**Comment by HowiePand** [comment]
Hi there, thanks for your reply. I modified the cik_lookup.py as described in master: [https://github.com/sec-edgar/sec-edgar/commit/eb5c3b8de8af70e8511f9e024cca89693cf7af1a.]

The "requests.exceptions.JSONDecodeError: Expecting value: line 1 column 1 (char 0)" Error disappears.

But When I run "my_filings.save('output')", a new error occurs:

my_filings.save('test')
C:\Users\Howie\anaconda3\lib\site-packages\bs4\builder\__init__.py:545: XMLParsedAsHTMLWarning: It looks like you're parsing an XML document using an HTML parser. If this really is an HTML document (maybe it's XHTML?), you can ignore or filter this warning. If it's XML, you should know that using an XML parser will be more reliable. To parse this document as XML, make sure you have the lxml package installed, and pass the keyword argument `features="xml"` into the BeautifulSoup constructor.
  warnings.warn(
Traceback (most recent call last):

  File "C:\Users\AppData\Local\Temp\ipykernel_48136\2582791835.py", line 1, in <module>
    my_filings.save('test')

  File "C:\Users\anaconda3\lib\site-packages\secedgar\core\company.py", line 340, in save
    loop.run_until_complete(self.client.wait_for_download_async(inputs))

  File "C:\Users\anaconda3\lib\asyncio\base_events.py", line 623, in run_until_complete
    self._check_running()

  File "C:\Users\anaconda3\lib\asyncio\base_events.py", line 583, in _check_running
    raise RuntimeError('This event loop is already running')

RuntimeError: This event loop is already running.

May I know how to solve it?  Thanks.

**Comment by jackmoody11** [comment]
Should be closed with #296. Master branch should fix this issue.
